src/main/resources/testnet.conf

```diff
@@ -91,6 +96,7 @@ scorex {
     nodeName = "ergo-testnet-5.0"
     nodeName = ${?NODENAME}
     knownPeers = [
+      "213.239.193.208:9022",
       "147.182.137.218:9022",
```

Comment:
Add the following nodes:
- 138.2.79.187:9052
- 192.234.196.165:9052

Reply:
:9022 you mean ?

Reply:
Yes he meant 9022 those are mine. I just realized that 138.2.79.187:9022 is NOT a real static, please replace that one with 168.138.185.215:9022 instead, that one will be static.

So add:  168.138.185.215:9022 + 192.234.196.165:9022

Reply:
ok, done